src/components/patientPage/patientCard/index.tsx

```diff
@@ -16,42 +16,27 @@ const PatientCard = ({ patient }: Props) => {
   getPatientNumberCard and getSubjectNumberCard are now rendering PatientInfo elements with click option which print the results on the console.
   */
   const getPatientNumberCard = (
-    object: "observations" | "conditions",
+    object:
+      | "observations"
+      | "conditions"
+      | "allergyIntolerances"
+      | "episodesOfCare",
     writtenName: string
   ) => {
+    let resourceNumber: string = "";
```

Comment:
I find it weird that resourceNumber is a string. Why don't you use a number here?

Reply:
And I think you can make it a const

Reply:
corrected, resourceNumber is now of type number and a const.